_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+
+## Public Land Survey System
+{: .text-left}
+
+I know, I know… that was probably more history than you were looking for, but really, that’s the annotated, crash-course for understanding the basis of the present-day Public Land Survey System (PLSS) (Fig 4). This system was originally proposed by Thomas Jefferson, and was mandated by Congress to oversee the cadastral surveys of the public lands. Essentially, it was set up to facilitate the transfer of Federal lands to private citizens. The Bureau of Land Management (BLM) is the official record keeper of the surveys. Over the past 200 years, almost 1.5 billion acres have been surveyed into townships (and ranges) and sections. These surveys were conducted primarily west of the original thirteen colonies and north of Texas (Texas has Spanish roots and Spanish land grants).  Presently, areas of Alaska are still being surveyed.
+
+<div class="caption">
+  <a href="{{ "/images/principal_meridians.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/principal_meridians_sm.png" | prepend: site.baseurl }}" alt="PLSS Principal Meridians">
+  </a>
+  <p class="caption-text">Fig 4: PLSS Principal Meridians</p>
+</div>
+<div class="caption">
+  <img src="{{ "/images/salt_lake_meridian.png" | prepend: site.baseurl }}" alt="Salt Lake Meridian" />
+  <p class="caption-text">Fig 5: Salt Lake Meridian</p>
+</div>
+
+As with any measuring system, you need a starting point. Within the PLSS, this is called the Principal Meridian. In Utah, this location is in Salt Lake City at Main Street and South Temple (Fig 5). All official surveys in Utah have an underlying reference to this point. In other words, every Township and Section in Utah is either North, South, East, or West of this location. Other PLSS states have their own unique principal meridians (Fig 4).
+
+The Townships are made up of six square miles, with each square mile known as a Section. These Sections are eventually subdivided down to an acre by way of Half Sections, Quarter Sections, Quarter-Quarter Sections, etc. (Fig 6). An acre, again, is simply 10 square chains - and there are 640 acres in a Section (a square-mile) or 40 acres in a quarter-quarter section. Hence the phrase, “40 acres and mule” is all you needed to be self-sufficient.
+
+<div class="caption">
+  <img src="{{ "/images/townships_sections.png" | prepend: site.baseurl }}" alt="Townships and Sections" />
+  <p class="caption-text">Fig 6: Townships, Ranges and Sections</p>
+</div>
+
+No matter how you shake it, though, land measurements in the West are consistently divisible by the magic 66 foot Gunter Chain. You’ll see its effect everywhere. This measuring unit was so successful because it made the math easy - and many early surveyors had a shaky grasp of mathematics and they were required to work quickly.
```

Comment:
```suggestion
No matter how you shake it, though, land measurements in the West are consistently divisible by the magic 66-foot Gunter Chain. You’ll see its effect everywhere. This measuring unit was so successful because it made the math easy---and many early surveyors had a shaky grasp of mathematics and were required to work quickly.
```